src/services/spinner.js

```diff
@@ -0,0 +1,99 @@
+const Spinnies = require('spinnies');
+
+const DEFAULT_SPINNER_NAME = 'default-spinner';
+
+class Spinner {
+  constructor() {
+    this.spinnies = new Spinnies({
+      spinnerColor: 'yellow',
+      spinner: {
+        interval: 80,
+        frames: ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'],
+        succeedPrefix: '✅',
+        failPrefix: '❌',
+      },
+    });
+    this.currentSpinnerOptions = null;
+  }
+
+  start(options) {
+    if (this.isRunning()) {
+      throw Error('A spinner is already running.');
+    }
+
+    this.currentSpinnerOptions = options;
+    this.spinnies.add(DEFAULT_SPINNER_NAME, options);
+  }
+
+  stop() {
+    if (!this.isRunning()) {
+      throw Error('No spinner is running.');
+    }
+
+    this.spinnies.stopAll();
+    this.currentSpinnerOptions = null;
+  }
+
+  success() {
```

Comment:
Judging from the doc, I think it's possible to add options here too (to update the text for instance).
```suggestion
  success(options = this.currentSpinnerOptions) {
```